build_tools/docker/manylinux2014_x86_64-release/Dockerfile

```diff
@@ -50,3 +50,16 @@ ENV BAZEL_LINKLIBS "-lstdc++ -lm"
 # fetch/build/install exactly what is needed.
 COPY install_tbb_manylinux2014.sh /usr/local/bin
 RUN bash /usr/local/bin/install_tbb_manylinux2014.sh
+
+######## AMD ROCM #######
+# Though the versions say latest it is known at the time of Docker compose.
+# Currently it is ROCM v5.2.1 and AMDGPU 22.20
+ARG ROCM_VERSION=latest
+ARG AMDGPU_VERSION=latest
```

Comment:
I'm not sure exactly what this comment means, but my understanding is that this would always fetch the latest when building the docker image. I'd prefer we pin the version and bump it as necessary. Otherwise if rebuilding the docker images for any reason where this layer can't be a cache hit it will potentially update the version. So anyone who updates the docker image has to deal with the consequences of a ROCM update.